src/components/brand-bar/BrandBar.stories.js

```diff
@@ -30,68 +30,59 @@ export default {
   },
 };
 
-const Template = (args) => ({
-  components: { UidsIowaBar },
-  setup() {
-    return { args }
-  },
-  template: `
-    <uids-iowa-bar :healthcare="args.healthcare" :narrow="args.narrow">
-      {{ args.default }}
-    </uids-iowa-bar>
-  `,
-})
+const Template = {
+  render: (args) => ({
+    components: { UidsIowaBar },
+    setup() {
+      return { args }
+    },
+    template: `
+      <uids-iowa-bar :healthcare="args.healthcare" :narrow="args.second_row_content || args.narrow">
+        <div v-if="args.second_row_content" class="parent-site-name">{{ args.default }}</div>
+        <template v-if="args.second_row_content" #second_row_content>
+          <h1 class="site-name">{{ args.second_row_content }}</h1>
+        </template>
+        <h1 v-else-if="args.default" class="site-name">{{ args.default }}</h1>
+      </uids-iowa-bar>
+    `,
+  }),
+};
 
-export const Default = Template.bind({})
-Default.args = {
-  narrow: false,
-  default: '',
-  second_row_content: '',
-  healthcare: false,
+export const Default = {
+  name: 'Default',
```

Comment:
This can be omitted because the name is the same as what would be automatically generated by Storybook (using the named export constant). We only need to set this when we want to override the default